README.md

```diff
@@ -25,74 +30,78 @@ reactivebleclient.scanForDevices(withService: uuid, scanMode: ScanMode.lowLatenc
   
 ```
 
-The `withservice` parameter is required.  The parameter `scanmode` is only used in Android and follows the conventions 
-described in [Android scansettings reference](https://developer.android.com/reference/android/bluetooth/le/ScanSettings). 
-If no scanmode is specified the library will use the balanced scanmode as reference.
+The `withService` parameter is required.  The parameter `scanMode` is only used on Android and follows the conventions described on [ScanSettings](https://developer.android.com/reference/android/bluetooth/le/ScanSettings#SCAN_MODE_BALANCED) Android reference page. If `scanMode` is omitted the balanced scan mode will be used.
 
 ### Establishing connection
 
 To interact with a device you first need to establish a connection:
 
 ```dart
 reactiveBleClient.connectToDevice(
-      id: 'AA:BB:CC:DD:EE:FF', 
```

Comment:
I removed all device IDs like this, because that suggest you can type something in. No, you can only get a valid device ID by performing a scan.

Reply:
makes sense is more clear what we mean indeed